plugins/modules/kafka_connectors.py

```diff
@@ -53,16 +53,21 @@
 '''
 
 import json
+import time
 
 from ansible.module_utils.basic import AnsibleModule
 from ansible.module_utils.urls import open_url
 import ansible.module_utils.six.moves.urllib.error as urllib_error
 __metaclass__ = type
 
+RUNNING_STATE = "RUNNING"
+WAIT_TIME_BEFORE_GET_STATUS = 1  # seconds
+TIMEOUT_WAITING_FOR_TASK_STATUS = 30  # seconds
+
 
 def get_current_connectors(connect_url):
     try:
-        res = open_url(connect_url)
+        res = open_url(connect_url, validate_certs=False)
```

Comment:
Another self doubt - are we having validate_certs set to False across the script to handle the case of ssl enabled?

Reply:
yes @utkarsh5474 , this is primarily to skip ssl certs verification in case of ssl enabled.
we'll re-visit this logic to pass certs and all when we deploy connectors with MTLS enabled.